setup.py

```diff
@@ -17,4 +17,9 @@
     "Operating System :: OS Independent",
   ],
   install_requires = ['requests', 'tqdm'],
+  entry_points={
+    "console_scripts": [
+        "aomame-translate=aomame.bin.translate:main",
+        ],
```

Comment:
Nice! I'll will use `click` but this is a good start to make the library CLI-able =)

Reply:
Let me add an issue to keep track of creating CLI able functions.